src/main/java/frc/utils/applicationsutils/OnDeploy.java

```diff
@@ -0,0 +1,21 @@
+package frc.utils.applicationsutils;
+
+import javax.swing.*;
+
+public class OnDeploy {
+
+
+    public static void main(String[] args) {
+        startupLogic();
+    }
+
+    private static void startupLogic() {
```

Comment:
```suggestion
    private static void startCompuerPrograms() {
```
or something in that direction